src/rev_ai/__init__.py

```diff
@@ -1,6 +1,6 @@
 # -*- coding: utf-8 -*-
 """Top-level package for rev_ai"""
 
-__version__ = '2.7.1'
+__version__ = '2.8.0'
```

Comment:
Do we need this version bump. The SDK itself did not change

Reply:
This version change is not to note a version specific to these changes. It is to set the correct version for the upcoming python release. The version was set ahead of the release. Other additions to this release should get a minor version bump instead of a patch version bump.